README.md

```diff
@@ -62,7 +62,8 @@ The values are split into two sections:
 | imagePullSecrets | list | `[]` | Image pull secrets. |
 | httpPort | int | `8080` | HTTP port. |
 | httpsPort | int | `8443` | HTTPS port. |
-| serviceType | string | `LoadBalancer` | Service type. |
+| service | object | `{ "enabled": true, "type": "LoadBalancer" }` | Service configuration. |
```

Comment:
why define the default object here, but not anywhere else?

Reply:
It's not a perfect science so far I'm afraid. I usually make a tradeoff based on how complicated (field count and level of nesting) the object is. This will largely become automatic when we transition to generating the readme from the `values.yaml` later (see #51).